### PR TITLE
Add print char

### PR DESCRIPTION
**Pull request by S-VIN**

### CODE REVIEW

**Review by nomad605dis — APPROVED** [review]
LGTM

File: src/TroykaOLED.cpp [hunk 1]
```diff
@@ -190,6 +184,12 @@ void TroykaOLED::print(String str, int x, int y) {
     _print(_codingCP866(data), x, y);
 }
 
+void TroykaOLED::print(char data, int x, int y) {
+    String s;
+    s += data;
+    print(s, x, y);
```
Comment (nkrkv) [inline src/TroykaOLED.cpp:195]: Oooh-oh. Looks ineffective. Why not low-level?

```cpp
void TroykaOLED::print(char ch, int x, int y) {
    char buff[2] = { ch, 0x00 };
    _print(_codingCP866(buff), x, y);
}
```

Comment (S-VIN) [reply src/TroykaOLED.cpp:195]: I checked your version. It's really more efficient than this one.

File: src/TroykaOLED.h [hunk 2]
```diff
@@ -160,6 +159,7 @@ class TroykaOLED {
     void print(int32_t n, int x = OLED_THIS, int y = OLED_THIS, uint8_t base = DEC);
     void print(uint32_t n, int x = OLED_THIS, int y = OLED_THIS, uint8_t base = DEC);
     void print(double n, int x = OLED_THIS, int y = OLED_THIS, uint8_t digits = 2);
+	void print(char ch, int x, int y);
```
Comment (nkrkv) [inline src/TroykaOLED.h]: Why moved here? Why with wrong indentation. Suggest putting it back where it was before the latest commit.

**Review by nkrkv — CHANGES_REQUESTED** [review]

File: src/TroykaOLED.h (hunk 2 above)
Comment (S-VIN) [reply src/TroykaOLED.h]: I didn't notice that notepad++ didn't replace tab with 4 spaces.

**Review by nkrkv — APPROVED** [review]
:ok_hand: LGTM

**Pull request merged**